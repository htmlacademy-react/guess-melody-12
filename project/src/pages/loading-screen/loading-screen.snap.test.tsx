import {render} from '@testing-library/react';
import LoadingScreen from './loading-screen';

describe('Component: LoadingScreen', () => {
  test('should render correctly', () => {
    const {container} = render(<LoadingScreen />);

    expect(container).toMatchSnapshot();
  });
});
