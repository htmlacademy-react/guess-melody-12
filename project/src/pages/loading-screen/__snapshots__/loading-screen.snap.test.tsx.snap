// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: LoadingScreen should render correctly 1`] = `
<div>
  <p>
    Loading ...
  </p>
</div>
`;
